packages/taxdat/R/setup_helpers.R

```diff
@@ -311,21 +311,11 @@ check_obs_model <- function(obs_model) {
 #' @return od_param if valid
 #' @export
 check_od_param <- function(obs_model, od_param) {
```

Comment:
Just to make sure, this function is no longer in use since we don't have `od_param` anymore, right? But of course, we could keep it.

Reply:
Correct we don't use this any more in this branch